TODO.md

```diff
@@ -6,7 +6,7 @@ Before 1.0
 ---
 
 Code:
-* [ ] Replace rm calls by rmDir in workflow 1.11
+* [x] Replace rm calls by rmDir in workflow 1.11
```

Comment:
Right. This file will be deleted during the release in any case, but I'll fix it